.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-9-release-golang-1.21-openshift-4.17
+  tag: rhel-9-release-golang-1.22-openshift-4.17
```

Comment:
the problem is that downstream CPO master is rebased on upstream release-1.29 which has Go 1.21 and k8s + deps on v1.29.1.

@mandre do you think we should bump the version of CPO to be on 1.30? The rest of OpenShift is already on 1.30.1: https://github.com/openshift/api/blob/master/go.mod#L10
so I think we would be safe?

https://github.com/kubernetes/cloud-provider-openstack/blob/release-1.29/go.mod#L3

Reply:
I proposed https://github.com/openshift/release/pull/53206

Reply:
That's only part of the fix. We'll need to rebase downstream CPO on upstream's `release-1.30` too.